QUICK_START.md

```diff
@@ -6,8 +6,12 @@ This document provides quick access to several different ways to set up CORTX fo
     1. On a single node on Kubernetes inside an OVA: [LINK](doc/ova/2.0.0/PI-5/CORTX_on_Open_Virtual_Appliance_PI-5.rst)
```

Comment:
@swatid-seagate, update the latest link as https://github.com/Seagate/cortx/blob/main/doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst

Reply:
Ok, sure!

Reply:
Done